WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -41,18 +42,50 @@ final class DashboardViewController: UIViewController {
         return label
     }()
 
+    /// A stack view to display `storeNameLabel` with additional margins
+    ///
+    private lazy var innerStackView: UIStackView = {
+        let view = UIStackView()
+        view.layoutMargins = UIEdgeInsets(top: 0, left: Constants.leadingMargin, bottom: 0, right: 0)
```

Comment:
I think it'd better if there's margin for the right inset too - this could cause missing margin for users using right-to-left languages like Arabic:
<img src="https://user-images.githubusercontent.com/5533851/127814699-03c2ed5e-b517-439f-8b09-918c76ee6d94.png" width=400 />

Reply:
Good catch, thanks! I added a margin for the right inset in c15b9fc74. I thought about using `directionalLayoutMargins` to add the inset to the leading edge specifically (not left/right) but I realized it would be good to have insets on both sides anyway in case of very long store names.